stdlib/grpc/build.gradle

```diff
@@ -73,6 +73,8 @@ dependencies {
     baloImplementation project(path: ':ballerina-crypto', configuration: 'baloImplementation')
     baloImplementation project(path: ':ballerina-time', configuration: 'baloImplementation')
     baloImplementation project(path: ':ballerina-http', configuration: 'baloImplementation')
+
+    interopImports project(':ballerina-config-api')
```

Comment:
Shall we keep all the inter-op imports at single place ?
(line 54 already has an inter-op import).